src/services/github_service.ts

```diff
@@ -0,0 +1,56 @@
+interface nestModule {
+  name: string;
+  normalizedName: string;
+  owner: string;
+  description: string;
+  repository: string;
+  latestVersion: string;
+  latestStableVersion: string;
+  packageUploadNames: string[];
+  locked: boolean | null;
+  malicious: boolean | null;
+  unlisted: boolean;
+  updatedAt: string;
+  createdAt: string;
+}
+
+export default class GitHubService {
+  /**
+   * Fetches the latest release of a a github repository
+   *
+   * @param githubUrl - Url of the github repository, eg https://github.com/drashland/dmm
+   *
+   * @returns The latest version.
```

Comment:
we going with @return or @returns? we should change this across all of our repositories lol

Reply:
I say returns?  everything we have is returns and it's just the `s` which isn't bold in the docs, still renders though

Reply:
alright